config/jobs/kubernetes/kops/build_jobs.py

```diff
@@ -973,12 +981,20 @@ def generate_upgrades():
         ((kops26, 'v1.24.0'), ('latest', 'v1.25.0')),
         ((kops26, 'v1.25.0'), ('latest', 'v1.25.0')),
         ((kops26, 'v1.25.0'), ('latest', 'v1.26.0')),
+        # 1.27 upgrade to latest
+        ((kops27, 'v1.24.0'), ('latest', 'v1.25.0')),
+        ((kops27, 'v1.24.0'), ('latest', 'v1.26.0')),
```

Comment:
This isn't a valid upgrade path, we can only upgrade one k8s minor version at a time. Same with a few others below

Reply:
It is just for us to know how what happens. We don't recommend it, but it may help catch bugs.